spec/controllers/spree/graphql_controller_spec.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+require 'spec_helper'
+
+RSpec.describe Spree::GraphqlController do
+  describe '#graphql' do
+    let(:action) { post '/graphql', params: params, headers: headers }
+    let(:params) { { } }
+    let(:headers) { { } }
+
+    context 'when the authentication is required' do
+      before do
+        Spree::GraphQL::Config.set(requires_authentication: false)
+        action
+      end
+
+      subject { response }
```

Comment:
What do you think about make the `post` request in the subject?

Reply:
@EduardoGHdez I was undecided about that, what is your reasoning about?

Reply:
If you execute your `post` in the subject, you will receive a `response` inmedialty. Without an implicit call.